ewkcoffea/modules/selection_Run3_2Lep.py

```diff
@@ -16,150 +11,111 @@
     "2022" : {
         "EGamma" : [
             "Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ",
+            "Ele23_Ele12_CaloIdL_TrackIdL_IsoVL",
+            "Ele30_WPTight_Gsf",
+            "Ele32_WPTight_Gsf",
+            "Ele32_WPTight_Gsf_L1DoubleEG",
+            "Ele35_WPTight_Gsf",
+            "Ele115_CaloIdVT_GsfTrkIdT",
+            "DoubleEle25_CaloIdL_MW",
+            "DoublePhoton70",
+        ],
+        "SingleMuon" : [
+            "IsoMu24",
+            "IsoMu27",
+            "Mu50",
+        ],
+        "DoubleMuon" : [
+            "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass3p8",
+            "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass8",
         ],
         "Muon" : [
             "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass3p8",
+            "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass8",
+            "IsoMu24",
+            "IsoMu27",
+            "Mu50",
         ],
         "MuonEG" : [
+            "Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ",
+            "Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL",
             "Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_DZ",
             "Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ",
         ]
     }
 
 }
 
-trgs_for_matching = {
-
-    "2022" : {
-        "m_m" : {
-            "trg_lst" : ["Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass3p8"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-        "e_e" : {
-            "trg_lst" : ["Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-        "m_e" : {
-            "trg_lst" : ["Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_DZ"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-        "e_m" : {
-            "trg_lst" : ["Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-    }
-}
-
 
 # Hard coded dictionary for figuring out overlap...
 #   - No unique way to do this
 #   - Note: In order for this to work properly, you should be processing all of the datastes to be used in the analysis
 #   - Otherwise, you may be removing events that show up in other datasets you're not using
 exclude_dict = {
-    "2022": {
```

Comment:
I think this exclude dict makes sense, but I think it would be good to add a comment to specify exactly what is going on. I.e. spell out what happened with single and double muon being merged to muon (include the run number) and explain why the ordering you have makes sense. That way it will be more clear to us in the future when we try to go back and understand what's happening.